lib/success.js

```diff
@@ -56,6 +58,10 @@ export default async function success(pluginConfig, context, { Octokit }) {
 
   if (successComment === false) {
     logger.log("Skip commenting on issues and pull requests.");
+    logger.error(`Issue and pull request comments should be disabled via 'successCommentCondition'.
+      Using 'false' for 'successComment' is deprecated and will be removed in a future major version.`);
```

Comment:
Hello @fgreinacher 👋, 

I am merely transferring this functionality/flow from the **GitLab** plugin to this place. But i'm a little bit lost on the deprecation talk of the `successComment and `failComment`, could you kindly shed some small light. 

🤔

Reply:
@babblebey Sure, the relevant discussion is here: https://github.com/semantic-release/gitlab/pull/730#issuecomment-2180348168

We decided to deprecate the old way of disabling comments and gently push folks to migrate to the new condition options.

Reply:
Thank you for the clarity @fgreinacher 💟

Reply:
Sure thing @babblebey, cool that you're picking it up so quickly!